demos/other/webglDOMLabels.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+        <title>VivaGraphs WebGL DOM Label test page</title>
+        <script src="../../dist/vivagraph.js"></script>
+
+        <script type='text/javascript'>
+            /*global Viva*/
+            function onLoad() {
+				var labelDom = document.getElementById('label');
+
+                var graphGenerator = Viva.Graph.generator();
+                var graph = graphGenerator.path(4);
+
+                var layout = Viva.Graph.Layout.forceDirected(graph, {
+                   springLength : 100,
+                   springCoeff : 0.0008,
+                   dragCoeff : 0.02,
+                   gravity : -1.2
+                });
+
+                var graphics = Viva.Graph.View.webglGraphics();
+
+                var renderer = Viva.Graph.View.renderer(graph, {
+                        layout     : layout,
+                        graphics   : graphics
+                    });
+
+                // I'm not quite happy with how events are currently implemented
+                // in the library and I'm planning to refactor it. But for the
+                // time beings this is how you track webgl-based input events:
+                var events = Viva.Graph.webglInputEvents(graphics, graph);
+
+                events.mouseEnter(function (node) {
+                    console.log('Mouse entered node: ' + node.id);
+
+                }).mouseLeave(function (node) {
+                    console.log('Mouse left node: ' + node.id);
+
+                }).dblClick(function (node) {
+                    console.log('Double click on node: ' + node.id);
+                }).click(function (node) {
+                    console.log('Single click on node: ' + node.id);
+                });
+
+                renderer.run();
+
+				function labelLoop () {
+					var node = graph.getNode(0)
+					var nodeUi = graphics.getNodeUI(node.id)
+					var pos = graphics.transformGraphToClientCoordinates({
+						x: nodeUi.position.x,
+						y: nodeUi.position.y
+					})
+
+					labelDom.style.left = pos.x + 'px'
+					labelDom.style.top = pos.y + 'px'
+
+					requestAnimationFrame(labelLoop)
+				}
+
+				requestAnimationFrame(labelLoop)
```

Comment:
I just realized this could be done via `placeNode()` callback.

I generalized your example a little bit. Please take a look: [Code](https://github.com/anvaka/VivaGraphJS/blob/6fa8973417e77d9ad44db9a9f60e52c85f337654/demos/other/webglDOMLabels.html#L28-L46)/[Live demo](https://anvaka.github.io/VivaGraphJS/demos/other/webglDOMLabels.html)